apps/storybook/static/locales/en/translation.json

```diff
@@ -0,0 +1,21 @@
+{
+  "Heading": "Heading",
+  "Some description": "Some description",
+  "What's included in the GitHub for Startups offer?": "What's included in the GitHub for Startups offer?",
+  "All GitHub for Startups companies receive up to 20 seats of GitHub Enterprise for free for year one and 50% off year two. Learn more about the features and capabilities of GitHub Enterprise": "All GitHub for Startups companies receive up to 20 seats of GitHub Enterprise for free for year one and 50% off year two. Learn more about the features and capabilities of GitHub Enterprise",
+  "here": "here",
+  "Who is eligible to apply?": "Who is eligible to apply?",
+  "Startups who meet the following criteria are eligible to apply for the program:": "Startups who meet the following criteria are eligible to apply for the program:",
+  "Must be associated with a current GitHub for Startups partner.": "Must be associated with a current GitHub for Startups partner.",
+  "Self-funded or funded (Seed-Series A)": "Self-funded or funded (Seed-Series A)",
+  "Not a current GitHub Enterprise customer": "Not a current GitHub Enterprise customer",
+  "Must not have previously received credits for GitHub Enterprise": "Must not have previously received credits for GitHub Enterprise",
+  "What if my startup is not eligible? Are there other resources for me?": "What if my startup is not eligible? Are there other resources for me?",
+  "If you're not currently eligible for the GitHub for Startups but would like to try GitHub Enterprise, please feel to sign up for a trial": "If you're not currently eligible for the GitHub for Startups but would like to try GitHub Enterprise, please feel to sign up for a trial",
+  "How can my organization become a GitHub for Startups partner?": "How can my organization become a GitHub for Startups partner?",
+  "Any investor, accelerator, or startup support organization is eligible to apply for the GitHub for Startups program.": "Any investor, accelerator, or startup support organization is eligible to apply for the GitHub for Startups program.",
+  "Apply here": "Apply here",
+  "Toggle color": "Toggle color",
+  "Description": "Description",
+  "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Blanditiis quidem veniam vero omnis consequuntur cum quae libero dolor dicta odio in, corporis perspiciatis nesciunt facere. Eius vero culpa quae itaque?": "Lorem ipsum dolor, sit amet consectetur adipisicing elit. Blanditiis quidem veniam vero omnis consequuntur cum quae libero dolor dicta odio in, corporis perspiciatis nesciunt facere. Eius vero culpa quae itaque?"
```

Comment:
A few of these appear to be unused. Is that intentional or can they be removed?

As an aside, is there any way to detect unused translations?

Reply:
I'll go through the list to double-check, but no there isn't a way to cross-check their usage. I have however, added a validator to ensure that keys are sync'd between the different translation files 😌